pkg/querier/http.go

```diff
@@ -119,6 +112,28 @@ func httpRequestToTailRequest(httpRequest *http.Request) (*logproto.TailRequest,
 	return &tailRequest, nil
 }
 
+func httpRequestToLookback(httpRequest *http.Request) (limit uint32, start, end time.Time, err error) {
```

Comment:
I would prefer a better name now that Lookback is not anymore existing.

Reply:
thought about this as well... However the return parameters describe exactly that... the specification of the Lookback window

Reply:
up to you.